hub/config/src/test/java/uk/gov/ida/hub/config/data/S3ConfigSourceTest.java

```diff
@@ -140,33 +141,29 @@ public void getRemoteConfigReturnsCachedConfigWhenRepeatedlyCalled() throws IOEx
     }
 
     @Test
-    public void getRemoteConfigOnlyRetrievesNewContentWhenLastModifiedChanges() throws IOException {
+    public void getRemoteConfigOnlyRetrievesNewContentWhenLastModifiedChanges2() throws Exception {
```

Comment:
why this becomes `2` ? i.e. getRemoteConfigOnlyRetrievesNewContentWhenLastModifiedChanges**2**()

Reply:
This was a hang over when we have the previous version of the test in place.  Now renamed.